tests/test.sh

```diff
@@ -5,10 +5,14 @@
 #
 set -e
 
-sudo DEBIAN_FRONTEND=noninteractive apt-get -qq update
-sudo DEBIAN_FRONTEND=noninteractive apt-get install -y -qq --no-install-recommends openjdk-8-jdk python3 python3-pip docker.io
+if [ -z ${NO_APT_INSTALL+x} ]; then
+ sudo DEBIAN_FRONTEND=noninteractive apt-get -qq update
+ sudo DEBIAN_FRONTEND=noninteractive apt-get install -y -qq --no-install-recommends openjdk-8-jdk python3 python3-pip docker.io
```

Comment:
pin versions? maybe we wanna ping ubuntu version to 20.04. We want to be aligned with all prod servers

Reply:
We're testing on Ubuntu 20.04 because that's what the CI offers, if it was easily accessible then I'd test on all. So our tests should handle the changes in new versions.

Stuff pulled into the gProfiler Docker image / executable should be pinned, to create a reproducible environment